std/traits.d

```diff
@@ -3550,6 +3551,176 @@ enum hasMember(T, string name) = __traits(hasMember, T, name);
 }
 
 /**
+ * Whether the symbol represented by the string, member, is a static member of T.
+ *
+ * Params:
+ *     T = Type containing symbol $(D member).
+ *     member = Name of symbol to test that resides in $(D T).
+ *
+ * Returns:
+ *     $(D true) iff $(D member) is static.
+ */
+template isStaticMember(T, string member)
+    if (__traits(hasMember, T, member))
+{
+    import std.meta : Alias;
+    alias sym = Alias!(__traits(getMember, T, member));
```

Comment:
It's not a DIP anymore, it'll be done without following the DIP process. The question is when.